### ISSUE TITLE
Replace "as of [DATE]" with clearer language

### ISSUE THREAD

**Issue by mthong** [codeforsanjose/economic-indicators-dashboard#84]
@juliemeridian points out that "as of" has ambiguous meaning. 

not sure what's a good replacement. 

options:
"updated [DATE]" ?
"last updated [DATE]" ? too wordy?

**Comment by LaurieSReynolds** [comment]
Just get rid of the text and list the date.  If there is just a year listed, can we get more precision?  Quarter or month?

**Comment by mthong** [comment]
there is no more precision when a year is listed. E.g. Census data is updated once a year. I think the data is collected over several months across the nation.

**Comment by mthong** [comment]
I think what I need is a better CSV editor than Excel. Excel does auto date formatting which is a pain for these files.

**Comment by mthong** [comment]
Removed the 'as of' and just listed the date.
